recipes/shadow3/conda_build_config.yaml

```
@@ -0,0 +1,9 @@
+c_compiler_version:            # [linux]
+  - 7                          # [linux]
+
+cxx_compiler_version:          # [linux]
+  - 7                          # [linux]
+
+fortran_compiler_version:  # [linux]
```

If both `linux` and `osx` values used to override the defaults, the block's selector should have the `unix` selector, otherwise the line `fortran_compiler_version:` won't be included in the config, resulting in a broken config.
```suggestion
fortran_compiler_version:  # [unix]
```